frontend/src/features/admin-form/create/payment/PaymentDrawer.tsx

```diff
@@ -330,11 +340,16 @@ export const PaymentDrawer = ({
   }, [paymentsField, resetData, setData])
 
   // Allows for payment data refresh in encrypt mode
-  if (!paymentData && isEncryptMode) return null
+  if (!paymentData && isEncryptMode && isStripeConnected) return null
+
+  const isDisabled = !(isEncryptMode && isStripeConnected) || isLoading
```

Comment:
should we factor the "payment enabled eligibility" check (i.e. `isEncryptMode && isStripeConnected`) out into it's own variable, since it's being reused a bunch?